Documentation/Registry.md

```diff
@@ -1667,6 +1679,12 @@ components:
         type: string
         enum:
           - - "1"
+    optionalContentVersion:
+      required: false
+      schema:
+        type: string
+        enum:
+          - - "1"          
```

Comment:
Why the double dash here? (Both here and in `contentVersion` above), I'd expect just

```yaml
enum:
  - "1"
```

Reply:
I am not sure. I thought it was some weird syntax that I didn't know about. Amended in https://github.com/apple/swift-package-manager/pull/6153/commits/69a4a024996b8c528e556ab905980dd7e871848b